en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
 gui.xaero_clear=Clear Set
 gui.xaero_from=From
 gui.xaero_to=To
-gui.xaero_used_seed=Seed To Use:
-gui.xaero_show_distance=Distance To WP
-gui.xaero_waypoints_scale=Ingame Waypoints Scale
-gui.xaero_block_colours=Block Colours
+gui.xaero_used_seed=Seed to Use:
+gui.xaero_show_distance=Distance to WP
+gui.xaero_waypoints_scale=In-Game Waypoint Scale
+gui.xaero_block_colours=Color Mode
 gui.xaero_accurate=Accurate
 gui.xaero_vanilla=Vanilla
 gui.xaero_lighting=Lighting
-gui.xaero_air_setting=Low Air Notification
-gui.xaero_air_low=Low Air
+gui.xaero_air_setting=Oxygen Notifications
+gui.xaero_air_low=Low Oxygen Value
 gui.xaero_looking_at=Looking At
-gui.xaero_all=All
-gui.xaero_display_redstone=Display Redstone
-gui.xaero_clear_set_message=Are you sure you would like to clear current set
-gui.xaero_clear_set_message2=All the waypoints will be deleted from the set.
+gui.xaero_all=All at Once
+gui.xaero_display_redstone=Show Redstone
+gui.xaero_clear_set_message=Are you sure you would like to clear the current set
+gui.xaero_clear_set_message2=All waypoints from this set will be deleted!
 gui.xaero_default=Default
 gui.xaero_create_set=New Waypoint Set
 gui.xaero_world=World
 gui.xaero_waypoint_set=Set
 gui.xaero_delete_set=Delete Set
-gui.xaero_delete_set_message=Are you sure you would like to delete current set
-gui.xaero_delete_set_message2=The set will be removed from current world.
-gui.xaero_toggle_waypoints=Toggle Ingame Waypoints
-gui.xaero_dots_scale=Entity Dots Scale
+gui.xaero_delete_set_message=Are you sure you would like to delete the current set
+gui.xaero_delete_set_message2=This set will be removed from the current world.
```

Comment:
This sentence has only period at the end, while a very similar one — `gui.xaero_clear_set_message2` — now has an exclamation mark. I decided not to replace the period here because this sentence is not a "critical" one. Waypoints are already non-existent at this point, so the only thing a user loses is an empty set. But perhaps I should have gone towards consistency...